server/command_list.go

```diff
@@ -7,19 +7,60 @@ import (
 	cloud "github.com/mattermost/mattermost-cloud/model"
 	"github.com/mattermost/mattermost-server/v6/model"
 	"github.com/pkg/errors"
+	flag "github.com/spf13/pflag"
 )
 
+type listConfig struct {
+	Shared bool
+}
+
+func getListFlagSet() *flag.FlagSet {
+	flagSet := flag.NewFlagSet("list", flag.ContinueOnError)
+	flagSet.Bool("shared-installations", false, "Lists shared installations instead of personal ones")
```

Comment:
This could be something like `-show (personal/shared/updates-allowed/all)` ??
Unsure if we would ever use the `all` thing.

Reply:
Great idea! I have similar thoughts that we may possibly add to the plugin in the future depending on usage.